### PR TITLE
Fixes enclave default for ERC20 contracts.

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

The enclave was defaulting to an empty list of ERC20 contract addresses. This would blow up when initialising the enclave, since at least two are expected to instantiate the bridge.

### What changes were made as part of this PR:

Functional.

- Enclave config defaults to the BTC and ETH ERC20 contract addresses
- Default config had to be moved to a different package, to avoid an import cycle (config -> evm -> wallet -> config)

### CODE REVIEW

File: go/obscuronode/enclave/enclaverunner/cli.go [hunk 1]
```diff
@@ -32,7 +34,12 @@ type EnclaveConfigToml struct {
 // ParseConfig returns a config.EnclaveConfig based on either the file identified by the `config` flag, or the flags
 // with specific defaults (if the `config` flag isn't specified).
 func ParseConfig() config.EnclaveConfig {
-	defaultConfig := config.DefaultEnclaveConfig()
+	defaultConfig := defaultconfig.DefaultEnclaveConfig()
```
Comment (tudor-malene) [inline go/obscuronode/enclave/enclaverunner/cli.go:37]: I find this name confusing. 
I would call this variable "config", and the one below :"eRC20ContractAddrs"

You start from the default as a baseline, basically make a copy of it, and then start populating the fields with what was passed in the config file.

File: go/obscuronode/config/defaultconfig/enclave_default_config.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		HostAddress:               "127.0.0.1:10000",
+		Address:                   "127.0.0.1:11000",
+		L1ChainID:                 1337,
+		ObscuroChainID:            777,
+		WillAttest:                false,
+		ValidateL1Blocks:          false,
+		GenesisJSON:               nil,
+		SpeculativeExecution:      false,
+		ManagementContractAddress: common.BytesToAddress([]byte("")),
+		ERC20ContractAddresses:    []*common.Address{&evm.WBtcContract, &evm.WEthContract},
```
[the remark this answers is not in the document]
Comment (tudor-malene) [reply go/obscuronode/config/defaultconfig/enclave_default_config.go:22]: Not sure what should be the default here, as these are parameters that are obtained after deploying  to the L1.
i suggest we use "common.Address{}"

**Review by tudor-malene — COMMENTED** [review]
some minor comments

File: go/obscuronode/enclave/enclaverunner/cli.go (hunk 1 above)
Comment (joeldudley) [reply go/obscuronode/enclave/enclaverunner/cli.go]: Sure, fixed.

File: go/obscuronode/config/defaultconfig/enclave_default_config.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (joeldudley) [reply go/obscuronode/config/defaultconfig/enclave_default_config.go]: Fixed

**Review by tudor-malene — APPROVED** [review]
lgtm.
The change now is that check to make sure there are 2 addresses passed in, and a clear error msg?

**Comment by joeldudley** [comment]
@tudor-malene Haha, yeah basically. Also clarified a couple of comments and renamed `defaultConfig`.

**Pull request merged**